src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,62 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <>
```

Comment:
you do not need this fragment here, you already have whapper with 'card' attribute

Reply:
checklist 1))

Reply:
of course!! there are many tags with no wrap

Reply:
thanks!